config/discourse_defaults.conf

```diff
@@ -147,6 +147,9 @@ redis_replica_port = 6379
 # redis database
 redis_db = 0
 
+# redis username
+redis_username = default
```

Comment:
Sorry one more request. Since this is optional, I think we can leave it blank instead of passing in `default` which I think will have a higher risk of leading to unexpected behaviours.

Reply:
Done